plugin/trino-memsql/src/main/java/io/trino/plugin/memsql/MemSqlClient.java

```diff
@@ -378,6 +382,46 @@ private static RemoteTableName getRemoteTable(ResultSet resultSet)
         return Optional.empty();
     }
 
+    public static ColumnMapping memsqlTimeColumnMapping(TimeType timeType) {
```

Comment:
Please ignore this WIP / exploratory attempt to fix singlestore wrapping LocalTime types.
I may close this PR + open a new one when it's ready - there's a lot of work still to do to clean this up and test it.
CC: @wendigo @hashhar